crates/sargon/src/profile/mfa/security_structures/security_shield_builder.rs

```diff
@@ -459,6 +506,19 @@ impl SecurityShieldBuilder {
     }
 }
 
+// ================
+// ==== STATIC ====
+// ================
+impl SecurityShieldBuilder {
+    pub fn sorted_factor_sources_for_selection(
```

Comment:
Isn't this specific for Select Factors for Transactions, aka for Access role? If so, then it should be specific. Also not all factor sources can be used for Access role, thus some filtering need to happen as well AFAIK.

Reply:
Security questions and trusted contact (not supported in MVP) cannot be used for Access role, we can filter them out

Reply:
correct, these should be filtered out accordingly.